tests/test_eval_model.py

```diff
@@ -33,11 +33,13 @@ def test_output(self):
         for i in range(1, len(scores)):
             score = ast.literal_eval(scores[i])
             # check totals
-            self.assertTrue(score['exs'] == i,
-                            "Total is incorrect")
+            self.assertEqual(score['exs'], i, "Total is incorrect")
             # accuracy should be one
-            self.assertTrue(score['accuracy'] == 1,
-                            "accuracy != 1")
+            self.assertEqual(score['accuracy'], 1, "accuracy != 1")
+            if 'rouge-1' in score:
+                self.assertEqual(score['rouge-1'], 1, 'rouge1 != 1')
+                self.assertEqual(score['rouge-2'], 0, 'rouge-2 != 0')
```

Comment:
? Rouge-1 and Rouge-L are 1.0, but Rouge-2 is 0?!!? That doesn't make much sense to me... Is the dataset just single words?

It might be better to add this test to something that uses the `-t integration_tests`, which uses 4-grams.

Reply:
Yes, this dataset are just {1, 2 ,3 }, do we need sth more complicated?